howso/direct/client.py

```diff
@@ -775,6 +775,9 @@ def create_trainee(  # noqa: C901
         # Initialize Amalgam entity
         self._initialize_trainee(trainee_id)
 
+        if self.configuration.client.client_extra_params.get("memory_validation", False):
```

Comment:
If we are defining this in the client extra params section, it should be a parameter to the client init method and captured there. Otherwise it should be a first class property of the client config class.